canonical-websites/www.ubuntu.com
# Update community.ubuntu.com links and redirects

## Done

* the new community site requires moving all deep links to the top level of the domain

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at:
  - [about](http://0.0.0.0:8001/about)
  - [server](http://0.0.0.0:8001/server)
  - [comm support](http://0.0.0.0:8001/support/community-support)
  - [contrib thank-you](http://0.0.0.0:8001/legal/contributors/thank-you)

etc…

## Issue / Card

Fixes #2265
